package NTrOverReqBundleIterator.NProto;

message TIndexFormPair {
    optional uint32 WordIndex = 1;
    optional uint32 FormClass = 2;
}

message TIndexFormPairs {
    repeated TIndexFormPair Pairs = 1;
}

message TFactoryData {
    optional bool UseConstraintChecker = 1;
    optional uint32 TrIteratorWordCount = 2;
    repeated TIndexFormPairs BlockTrIteratorWordIdxs = 3;
    repeated bool BlockMustNot = 4;
}
